pkg/virt-operator/resource/apply/core.go

```diff
@@ -320,20 +397,33 @@ func (r *Reconciler) createOrUpdateComponentsWithCertificates(queue workqueue.Ty
 		return err
 	}
 
-	// create/update CA Certificate secret
+	// create/update export CA Certificate secret
 	caExportCert, err := r.createOrUpdateCACertificateSecret(queue, components.KubeVirtExportCASecretName, caExportDuration, caExportRenewBefore)
 	if err != nil {
 		return err
 	}
 
+	err = r.createExternalKubeVirtCAConfigMap(findRequiredCAConfigMap(components.ExternalKubeVirtCAConfigMapName, r.targetStrategy.ConfigMaps()))
+	if err != nil {
+		return err
+	}
+
+	log.Log.V(3).Info("reading external CA configmap")
+	externalCACerts := r.getRemotePublicCas()
+	log.Log.V(3).Infof("found %d external CA certificates", len(externalCACerts))
 	// create/update CA config map
-	caBundle, err := r.createOrUpdateKubeVirtCAConfigMap(queue, caCert, caRenewBefore, findRequiredCAConfigMap(components.KubeVirtCASecretName, r.targetStrategy.ConfigMaps()))
+	caBundle, err := r.createOrUpdateKubeVirtCAConfigMap(queue, caCert, externalCACerts, caRenewBefore, findRequiredCAConfigMap(components.KubeVirtCASecretName, r.targetStrategy.ConfigMaps()))
+	if err != nil {
+		return err
+	}
+
+	err = r.cleanupExternalCACerts(findRequiredCAConfigMap(components.ExternalKubeVirtCAConfigMapName, r.targetStrategy.ConfigMaps()))
```

Comment:
Do I understand correctly that this is supposed to wipe the external CA ConfigMap after it was read by virt-operator? Why not simply delete the CM at this point? Also, the code is quite similar to `createExternalKubeVirtCAConfigMap` IIUC?

Reply:
For my understanding, once a CA was added to KV's CA ConfigMap it is cleared from this CM and new CAs can be added through it? Is there a way to ever remove CAs from KV's CA ConfigMap again?

Reply:
They get removed automatically when they expire.

Reply:
> Do I understand correctly that this is supposed to wipe the external CA ConfigMap after it was read by virt-operator? Why not simply delete the CM at this point? Also, the code is quite similar to `createExternalKubeVirtCAConfigMap` IIUC?

That is also a possibility, but then I have to recreate it as well. It was simpler to just wipe the contents.